pkgs/development/python-modules/django-modeltranslation/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, django
+, pytestCheckHook
+, pytest-cov
+, pytest-django
+, parameterized
+}:
+let
+  # 0.18.12 was yanked from PyPI:
+  # <https://pypi.org/project/django-modeltranslation/0.18.12/>
+  version = "0.18.11";
+in
+buildPythonPackage {
+  pname = "django-modeltranslation";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "deschler";
+    repo = "django-modeltranslation";
+    rev = "v${version}";
+    hash = "sha256-WEtTy449z7Fo9+UmiM+QAuUJ5eQ1RFe1HrIqFrY3L9k=";
+  };
+
+  disabled = pythonOlder "3.6";
+
+  propagatedBuildInputs = [ django ];
+
+  nativeCheckInputs = [ pytestCheckHook pytest-cov pytest-django parameterized ];
+
+  # I would try to remove `pytest-cov` completely but it is simpler this way
+  pytestFlagsArray = [ "--no-cov" ];
```

Comment:
please use substitueinplace to remove these cov flags:
https://github.com/deschler/django-modeltranslation/blob/master/pytest.ini#L6-L8

Reply:
Done!